rpcs3/rpcs3qt/gs_frame.cpp

```diff
@@ -363,7 +363,7 @@ void gs_frame::close()
 
 		if (!Emu.IsStopped())
 		{
-			Emu.Stop();
+			Emu.GracefulShutdown();
```

Comment:
this leads to a crash, i haven't looked into it yet.

Reply:
The crash is caused by the non-blocking nature of GracefulShutdown.
It is fixed by using the same while loop as you already did in MainWindow closeEvent, including the ProcessEvents call.

Reply:
```suggestion
			Emu.GracefulShutdown();

			while (!Emu.IsStopped())
			{
				QApplication::processEvents();
				std::this_thread::sleep_for(200ms);
			}
```

Reply:
Hopefully fixed, haven't tested. I don't know what are the consequences of executing processEvents "casually" at GracefulShutdown.